### DISCUSSION TITLE
在编辑时时常出错（如果开启了chrome翻译的情况下）

### DISCUSSION THREAD

**Discussion by kkk-gitbook** [GitbookIO/community#641] [0. General bugs and questions]
![image](https://github.com/GitbookIO/community/assets/172463681/5e492096-f1a0-4e52-b034-9e644d4a4609)

![image](https://github.com/GitbookIO/community/assets/172463681/3ee25ab7-4f42-4a6a-b9cc-be29e9ec5e87)

如果没有开启翻译，则不会出错。

**Comment by aikoii** [comment]
@kkk-gitbook This is a good shoutout indeed! I've added this to our backlog for future improvements.
